scripts/codegen/CAN/canrx_codegen.py

```diff
@@ -232,9 +266,15 @@ def __init_functions(self, function_prefix):
                     msg_uppercase_name=msg.snake_name.upper())
 
             signal_setters_fmt = '''
-            App_CanRx_{msg_uppercase_name}_SetSignal_{signal_name_uppercase}(
+            App_CanRx_{msg_uppercase_name}_SetPeriodicSignal_{signal_name_uppercase}(
```

Comment:
To make the `_add_critical_fault` cleaner. I split `CanRx_SetSignal` up to become  `CanRx_SetPeriodicSignal` and `CanRx_SetNonPeriodicSignal` to it matches `CanTx_SetPeriodicSignal` better. That's the reason for all the changes in this script.